services/app/src/components/chessBoard/chess-board.css

```diff
@@ -57,116 +25,79 @@ table {
 }
 
 #chess-board .border-tile:hover, .non-border-tile:hover {
-  border: 1px solid #6547FD;
-  width: 70px;
-  height: 70px;
+  border: 1px solid var(--game-light-purple);
+}
+
+#chess-board .border-tile,
+#chess-board .non-border-tile {
+  aspect-ratio: 1 / 1;
 }
 
 #chess-board .is-empty-tile:hover::before {
   content: "";
-  width: 40px;
-  height: 40px;
-  background: #6547FD;
+  width: 60%;
+  aspect-ratio: 1 / 1;
+  background: var(--game-light-purple);
   opacity: 0.5;
-  border-radius: 40px;
+  border-radius: 50%;
   position: absolute;
   transform: translateX(-50%) translateY(-50%);
   top: 50%;
   left: 50%;
 }
 
-#chess-board .attribute {
-  display: flex;
-  justify-content: center;
-  flex-direction: column;
-  padding: 8px;
-}
-
 #chess-board .attribute .index {
-  font-family: 'Lato';
-  font-style: normal;
-  font-weight: 700;
-  font-size: 20px;
   line-height: 28px;
   letter-spacing: -0.15px;
   color: var(--primary-text);
   text-transform: uppercase;
 }
 
 #chess-board .attribute .points {
-  font-family: 'Rubik';
-  font-style: normal;
-  font-weight: 500;
-  font-size: 11px;
   line-height: 16px;
   letter-spacing: 0.5px;
   text-transform: uppercase;
   color: var(--secondary-text);
 }
 
 #chess-board .square {
-  display: flex;
   position: relative;
-  height: 70px;
-  width: 70px;
-  margin: auto;
-  flex-direction: column;
-  justify-content: center;
+  height: clamp(59px, 6.6vh,70px);
+  aspect-ratio: 1 / 1;
+  object-fit: contain;
 }
 
-#chess-board .square .bubble-container {
-  display: flex;
-  justify-content: center;
-  margin-bottom: 2px;
-}
-
-#chess-board .square .name {
-  font-family: 'Poppins';
-  font-style: normal;
-  font-weight: 500;
-  font-size: 16px;
-  line-height: 16px;
-  text-transform: uppercase;
-  color: var(--primary);
-}
-
-#chess-board .square .figure-container {
+.square-move-info {
   display: flex;
+  flex-direction: column;
   justify-content: space-around;
-  padding: 0px 3px
+  align-items: center;
+  height: 100%;
 }
 
-#chess-board .square .figure-container-centered {
-  justify-content: center;
+.square-move-info .square-avatar-container {
+  height: 45%;
+  z-index: 2;
 }
 
-#chess-board .square .figure-img {
-  height: 23px;
+.square-move-info .square-move-text {
+  height: 25%;
+  z-index: 2;
 }
 
-#chess-board .square .figure-img-nth {
-  margin-left: -4px;
+#chess-board .square .figure-img {
+  height: clamp(16px, 2vh,23px);
```

Comment:
isn't parent sqaure `clamp` enough?

Reply:
Its possible to do a constant height for it, but then it looks kind of inconsistent. If you think about setting height in percentages, then it does not work as intended.

Reply:
Changed the height to 20px